corehq/toggles/__init__.py

```diff
@@ -123,7 +123,7 @@ class StaticToggle(object):
     def __init__(self, slug, label, tag, namespaces=None, help_link=None,
                  description=None, save_fn=None, enabled_for_new_domains_after=None,
                  enabled_for_new_users_after=None, relevant_environments=None,
-                 notification_emails=None):
+                 notification_emails=None, dependent_toggles=None):
```

Comment:
nit: maybe `dependent_on_toggles`, `dependency_toggles`, or `parent_toggles` to make the directionality clearer?

Reply:
91f7ed32b8f